charts/kvisor/values.yaml

```diff
@@ -94,17 +94,11 @@ agent:
       type: spc_t
     readOnlyRootFilesystem: true
 
-  resources: {}
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  resources:
```

Comment:
We need to discuss this IMHO. We are creating a critical component, which means we should have fixed memory and CPU requests and limits. Also they should match, as this gives the most predictable outcome. This is discussion is too big for this PR though.